### PR TITLE
pytest: prefer pytest executable without dot

### PR DESCRIPTION
**Pull request by blueyed**

### CODE REVIEW

File: autoload/test/python/pytest.vim [hunk 1]
```diff
@@ -7,7 +7,7 @@ function! test#python#pytest#test_file(file) abort
     if exists('g:test#python#runner')
       return g:test#python#runner == 'pytest'
     else
-      return executable('py.test')
+      return executable('pytest') || executable('py.test')
```
Comment (blueyed) [inline autoload/test/python/pytest.vim]: Not sure if keeping the 2nd one makes sense?!

Comment (janko-m) [reply autoload/test/python/pytest.vim]: If `pytest` will always exist alongside `py.test`, then by all means remove the `py.test` check.

**Comment by janko-m** [comment]
I pushed your commit to master with tests added, and modified the logic to still support `py.test` when using versions 1.x and 2.x; as far as I could tell 3.x starts using the `pytest` executable name.

**Comment by blueyed** [comment]
Thanks!

**Pull request closed without merging**